### DISCUSSION TITLE
Interacting with an Anvil instance

### DISCUSSION THREAD

**Discussion by gershido** [wagmi-dev/anvil.js#33] [Q&A] [answered]
Hey :) I'm trying to figure out how to call `evm_increaseTime` programmatically from a test (using Jest). So far can't find any convenient way to do so. Any help? 
Thank you!

**Comment by fubhy** [comment, marked as answer]
Your best bet would be to create a TestClient with viem (https://viem.sh/docs/clients/test.html).

Check out the examples here for instance: https://github.com/wagmi-dev/anvil.js/blob/main/examples/example-vitest/tests/utils.ts#L15-L38

There's no `increaseTime` in those examples, but there's a `testClient.reset()` call here: https://github.com/wagmi-dev/anvil.js/blob/main/examples/example-vitest/tests/setup.ts#L8-L11

This should all work anaologues in Jest.
